tests/Event/ApiGateway/WebsocketEventTest.php

```diff
@@ -0,0 +1,92 @@
+<?php declare(strict_types=1);
+
+namespace Bref\Test\Event\DynamoDb;
+
+use Bref\Event\ApiGateway\WebsocketEvent;
+use Bref\Event\ApiGateway\WebsocketResponse;
+use PHPUnit\Framework\TestCase;
+
+class WebsocketEventTest extends TestCase
+{
+    public function test_connect()
+    {
+        // Arrange
```

Comment:
Could you remove these comments? (these would not be inline with the rest of the codebase)

Keep the practice however, that's in general how the rest of the tests are organized.

Reply:
Sure. Actually copied over the test from DynamoDbEventTest.php - so I thought it was standard practice in the code-base and left them. I'll remove them.

Reply:
Right sorry I probably missed that in the DynamoDB PR, thanks!